frontend/controllers/IndexController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace frontend\controllers;
+
+use Yii;
+use yii\base\InvalidParamException;
+use yii\web\BadRequestHttpException;
+use yii\web\Controller;
+use yii\filters\VerbFilter;
+use yii\filters\AccessControl;
+use common\models\LoginForm;
+use frontend\models\PasswordResetRequestForm;
+use frontend\models\ResetPasswordForm;
+use frontend\models\SignupForm;
+use frontend\models\ContactForm;
+
+/**
+ * Site controller
+ */
+class IndexController extends Controller
+{
+    /**
+     * @inheritdoc
+     */
+    
+    /**
+     * @inheritdoc
+     */
+    public function actions()
+    {
+        return [
+            'error' => [
+                'class' => 'yii\web\ErrorAction',
+            ],
+            'captcha' => [
+                'class' => 'yii\captcha\CaptchaAction',
+                'fixedVerifyCode' => YII_ENV_TEST ? 'testme' : null,
+            ],
+        ];
+    }
+
+    /**
+     * Displays homepage.
+     *
+     * @return mixed
+     */
+    public function actionIndex()
+    {
+        //echo "!!!!MY TEST INDEX ACTION!!!!";
```

Comment:
Please remove this if it is not needed. By the way you could return your response here instead of echoing the text, like:
```
Yii::$app->response->content = 'hello world!';
```
Please refer to http://www.yiiframework.com/doc-2.0/guide-runtime-responses.html